server/api.js

```diff
@@ -17,4 +20,68 @@ router.get("/", (_, res, next) => {
 	});
 });
 
+router.get("/questions",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("questions");
+		collection.find().toArray((err, questions)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(questions);
+			}
+		});
+	});
+});
+
+router.post("/question",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("questions");
+		const question = req.body;
+
+		collection.insertOne(question,(err, question)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(question);
+			}
+		});
+	});
+});
+
+router.post("/quiz",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("quizes");
+		const question = req.body;
+
+		collection.insertOne(question,(err, quiz)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(quiz);
+			}
+		});
```

Comment:
LOOKS GOOD, HAVE YOU TEST IT?

Reply:
Yes, I tested it, but I didn't any validation for req.body

Reply:
@naderakhgari @sabia-m this might help with validating the request before inserting into the database
https://express-validator.github.io/docs/